microsite/docs/pull_request.md

```diff
@@ -251,7 +255,98 @@ The `result` on the right is the created [PullRequestReview][pr-scala].
 
 See [the API doc](https://developer.github.com/v3/pulls/reviews/#create-a-review-for-a-pull-request) for full reference.
 
+## Review requests
+
+This API allows you to operate review requests. Reviewers could be both users and teams. Usernames should be used without a leading '@' sign.
+**NOTE**: There might be problems with team reviewers and Github App authentication. 
```

Comment:
can you expand a bit on that? what's the issue?

Reply:
Got errors trying to request a review from a team via github app auth. On a second thought, that most likely was a misconfiguration on the app's permissions, so I'd rather drop that note.